### PR TITLE
MINOR: Use the correct label while inspecting the docker images.

### PR DESCRIPTION
**Pull request by kamalcph**
- Get a valid return code from the PIPESTATUS command.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by kamalcph** [comment]
When using PIPESTATUS, the result should be cached in an array. Otherwise, the new commands will replace the result in it: https://www.shellscript.sh/tips/pipestatus/

File: tests/docker/ducker-ak [hunk 1]
```diff
@@ -311,15 +313,16 @@ use only ${num_nodes}."
 
     ducker_build "${image_name}"
 
-    docker inspect --format='{{.Config.Labels}}' --type=image "${image_name}" | grep -q 'ducker.type'
-    local docker_status=${PIPESTATUS[0]}
-    local grep_status=${PIPESTATUS[1]}
+    docker inspect --format='{{.Config.Labels}}' --type=image "${image_name}" | grep -q 'ducker.creator'
```
Comment (tombentley) [inline tests/docker/ducker-ak:339]: Why change the argument to grep?

File: tests/docker/ducker-ak [hunk 2]
```diff
@@ -217,6 +217,8 @@ ask_yes_no() {
 # $1: The name of the image to build.
 ducker_build() {
     local image_name="${1}"
+    local what="${2}"
+    [[ -n "${what}" ]] || what="${image_name}"
```
Comment (tombentley) [inline tests/docker/ducker-ak]: The new argument isn't mentioned in the PR description and isn't used in the rest of the patch.

File: tests/docker/ducker-ak (hunk 1 above)
Comment (kamalcph) [reply tests/docker/ducker-ak:339]: `ducker.creator` is the label used in the [Dockerfile](https://github.com/apache/kafka/blob/db688b1a5e5904eed5849fbab4940585e5a3d646/tests/docker/Dockerfile#L32).

File: tests/docker/ducker-ak (hunk 2 above)
Comment (kamalcph) [reply tests/docker/ducker-ak]: The `what` argument is used when printing the [status](https://github.com/apache/kafka/blob/db688b1a5e5904eed5849fbab4940585e5a3d646/tests/docker/ducker-ak#L241) of the docker build.

**Comment by kamalcph** [comment]
@cmccabe @tombentley 
Please take a look

**Pull request closed without merging**